tritonbench/operators/flex_attention/operator.py

```diff
@@ -14,110 +31,234 @@
     register_x_val,
 )
 
-# Default configurations for the benchmark
-BATCH_SIZE = 8
-NUM_HEADS = 16
-SEQ_LEN = 1024
-HEAD_DIM = 128
-DTYPE = torch.bfloat16
+try:
+    import attn_gym  # noqa: F401
+except ImportError as e:
+    print(
+        "attn_gym not found, to install run `pip install git+https://github.com/pytorch-labs/attention-gym.git`"
+    )
+    raise e
+
+# TODO how should I enable these
```

Comment:
@xuzhao9 Any thoughts on this one, I didn't want to have these settings pollute other entries but this is important since there are a lot of recompiles

Reply:
By default we will isolate each operator in its own process, so enabling these options will only affect this operator `flex_attention`.